content/05.administrative.md

```diff
@@ -5,6 +5,8 @@ This work was funded through the Alex’s Lemonade Stand Foundation (ALSF) Child
 This project has been funded in part with Federal funds from the National Cancer Institute, National Institutes of Health, under Contract No. 75N91019D00024, Task Order No. 75N91020F00003 (JLR, ACR, APH). 
 The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products or organizations imply endorsement by the U.S. Government.
 
+The authors would like to thank the following collaborators who contributed to or supervised contributions to the analysis repository, but did not qualify for authorship: William Amadio, Holly Beale, Ellen Kephart, Alfred Lyle, and Olena Vaske.
```

Comment:
I might add modify this sentence to explicitly refer to the authorship guidelines we are using, e.g. `did not qualify for authorship based on [guidelines](link)`? That way it doesn't seem like we may have arbitrarily denied authorship to contributors/supervisors.